temporian/implementation/numpy/operators/core_mapping.py

```diff
@@ -1,9 +1,13 @@
-from temporian.implementation.numpy.operators.assign import NumpyAssignOperator
+from temporian.implementation.numpy.operators.assign import (
+    AssignNumpyImplementation,
+)
 from temporian.implementation.numpy.operators.select import NumpySelectOperator
-from temporian.implementation.numpy.operators.sum import NumpySumOperator
+from temporian.implementation.numpy.operators.arithmetic import (
+    ArithmeticNumpyImplementation,
+)
 
 OPERATOR_IMPLEMENTATIONS = {
     "SELECT": NumpySelectOperator,
```

Comment:
We should rename this one to `SelectNumpyImplementation` to keep consistency with the others

Reply:
I think I should do this on another PR and change `SelectNumpyImpelemntation` accordingly. It could be made on #28 or a new PR